e2e-tests/functions

```diff
@@ -572,6 +628,57 @@ retry() {
     done
 }
 
+start_vault() {
+    name=${1:-vault-service}
+    create_namespace "$name"
+    deploy_helm "$name"
+    helm delete --purge "$name" || :
+    
+    desc "install Vault $name"
+    helm install --name $name --namespace "$name" \
+        --set dataStorage.enabled=false \
+        "$src_dir/vendor/vault"
+
+    set +o xtrace
+    retry=0
+    echo -n pod/$name-0
+    until kubectl get pod/$name-0 -o 'jsonpath={.status.containerStatuses[0].state}' 2>/dev/null | grep 'running'; do
+        echo -n .
+        sleep 1
+        let retry+=1
+        if [ "$retry" -ge 480 ]; then
+            kubectl_bin describe pod/$name-0
+            kubectl_bin logs $name-0
```

Comment:
```suggestion
    until kubectl get --namespace="$name" pod/$name-0 -o 'jsonpath={.status.containerStatuses[0].state}' 2>/dev/null | grep 'running'; do
        echo -n .
        sleep 1
        let retry+=1
        if [ "$retry" -ge 480 ]; then
            kubectl_bin describe --namespace="$name" pod/$name-0
            kubectl_bin logs --namespace="$name" $name-0
```

Reply:
We do not need all these `--namespace` flags because this code runs in vault namespace.
When we execute `create_namespace` function it automatically put us to this namespace. And I think it's good because when vault is starting I want to be able to execute  `kubeclt get ...` without specifying namespace (which name I have to find first).
In my daily development routine it's common practice to run `watch -n 1 kubeclt get all` in separate console tab to see what's going on. So this is one more point I like that `create_namespace` switches to just created namespaces.